src/AccessibilityInsights.CommonUxComponents/CommonUxComponents.csproj

```diff
@@ -8,29 +8,6 @@
 
   <Import Project="..\..\build\NetFrameworkRelease.targets" />
 
-  <ItemGroup>
```

Comment:
These are all in NetFrameworkRelease.targets, so they don't need to be duplicated in other projects. As shown with https://github.com/microsoft/accessibility-insights-windows/pull/894/files, dependabot is able to find and updated references in the .targets file